### ISSUE TITLE
alt-text on images is missing

### ISSUE THREAD

**Issue by abbycabs** [github/maintainermonth#65]

**Comment by abbycabs** [comment]
@david-mbugua Here's a separate issue for alt text!

**Comment by david-mbugua** [comment]
@abbycabs Oh nice. I will work on it accordingly. Thanks

**Comment by juliaferraioli** [comment]
Alt-text won't show in the UI -- you'll have to go into developer tools to see if it's being populated. An element with alt-text, once populated, will have the `alt=""` tag filled with something when you look at the HTML.

**Comment by david-mbugua** [comment]
Oh yeah, I understand that. It's not being populated in the Developer tools as it should even after I add the alt text. KIndly look at lines 7 & 18 on the Example.jsx and lines 27 & 30 on GetInvolved.jsx below. Kindly assist :)
<img width="1680" alt="Screenshot 2023-03-18 at 00 24 30" src="https://user-images.githubusercontent.com/82040907/226056459-b46d929b-8bf2-4c0c-923f-de1000dc2f06.png">
<img width="1680" alt="Screenshot 2023-03-18 at 00 24 17" src="https://user-images.githubusercontent.com/82040907/226056566-d157521e-abd0-455f-bd3a-0caaeb00bab2.png">

**Comment by abbycabs** [comment]
Thanks David, I can take a look next weekOn Mar 17, 2023, at 5:30 PM, David Mbugua ***@***.***> wrote:﻿Re: [github/maintainermonth] alt-text on images is missing (Issue #65)Oh yeah, I understand that. It's not being populated in the Developer tools as it should even after I add the alt text. KIndly look at lines 7 & 18 on the Example.jsx and lines 27 & 30 on GetInvolved.jsx below. Kindly assist :)—Reply to this email directly, view it on GitHub or unsubscribe.You are receiving this email because you authored the thread.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by david-mbugua** [comment]
That's well noted and thank you.

**Comment by david-mbugua** [comment]
Hi @abbycabs , Did you manage to take a look at the above issue?

**Comment by abbycabs** [comment]
@david-mbugua not yet, I might have some time tomorrow.
In the meantime, if you have alt-text you plan on using for each image, feel free to drop them in this issue.

**Comment by david-mbugua** [comment]
@abbycabs  These are the alt-text for the images in the 'get involved section'

```
examples:
  - socialMedia:
      - title: 'Posting on social media to thank the maintainers of their favorite open source projects'
      - image: '/images/thanks.png'
        alt: 'Illustration of a person thanking an open source maintainer on social media' 
      - imageDescription: 'Illustration'
  - podcasts:
      - title: 'Recording special podcast episodes'
      - image: '/images/podcast.png'
        alt: 'Illustration of a person recording a podcast episode about open source maintainers'
      - imageDescription: 'Illustration'
  - meetings:
      - title: 'Organizing special-edition open working group meetings'
      - image: '/images/meetings.png'
        alt: 'Illustration of people at an open working group meeting for open source maintainers'
      - imageDescription: 'Illustration'
  - miniEvents:
      - title: 'Hosting mini-events'
      - image: '/images/events.png'
        alt: 'Illustration of people hosting a mini-event for open source maintainers'
      - imageDescription: 'Illustration'
  - summits:
      - title: 'Convening tool-specific maintainer summits'
      - image: '/images/tools.png'
        alt: 'Illustration of people at a tool-specific maintainer summit for open source projects'
      - imageDescription: 'Illustration'
  - streaming:
      - title: 'Streaming live on Twitch'
      - image: '/images/twitch.png'
        alt: 'Illustration of a person streaming on Twitch about open source maintainers'
      - imageDescription: 'Illustration'
  - donations:
      - title: 'Pledging to donate to support Open Source'
      - image: '/images/donate.png'
        alt: 'Illustration of a person making a donation to support open source'
      - imageDescription: 'Illustration'
```

**Comment by abbycabs** [comment]
Hi @david-mbugua - I took a look and the imageDescription attribute wasn't being read it for the examples.[ I made a PR to add it in! ](https://github.com/github/maintainermonth/pull/66/files)Once it's merged you should be able to update `imageDescription` in `/content/home/3-get-involved.md` to display alt-text on these images. No need to add a separate `alt` attribute.

Looking forward to your PR!

**Comment by david-mbugua** [comment]
Hi @abbycabs Thank you so much. I am looking forward to making this contribution. 

Thanks

**Comment by david-mbugua** [comment]
Hi @abbycabs I did the PR on 25th. Did you see it?

**Comment by abbycabs** [comment]
Thanks @david-mbugua! Yes I saw it come in over the weekend. Just requested some changes.

**Comment by abbycabs** [comment]
These changes are live! Thanks @david-mbugua 

https://github.com/github/maintainermonth/pull/69
